tests/test_broadcast_area.py

```diff
@@ -1,83 +1,68 @@
 import pytest
-from json import JSONDecodeError
-from unittest import mock
+import re
 
 from notifications_utils.broadcast_areas import (
-    BroadcastAreaLibraries,
+    BroadcastAreasRepository,
     broadcast_area_libraries,
-    load_geojson_file,
 )
 
 
 def test_loads_libraries():
     assert [
-        (library.id, library.name) for library in sorted(broadcast_area_libraries)
+        (library.id, library.name, library.is_group) for library in sorted(broadcast_area_libraries)
     ] == [
         (
             'counties-and-unitary-authorities-in-england-and-wales',
-            'Counties and Unitary Authorities in England and Wales'),
+            'Counties and Unitary Authorities in England and Wales',
+            False,
+        ),
         (
             'countries',
             'Countries',
+            False,
+        ),
+        (
+            'electoral-wards-of-the-united-kingdom',
+            'Electoral Wards of the United Kingdom',
+            True,
         ),
         (
             'regions-of-england',
             'Regions of England',
+            False,
         ),
     ]
 
 
-@pytest.mark.xfail(raises=JSONDecodeError)
-def test_raises_for_invalid_json():
-    load_geojson_file.cache_clear()
-    with mock.patch(
-        'notifications_utils.broadcast_areas.Path.read_text',
-        return_value='foo',
-    ):
-        BroadcastAreaLibraries()
-
-
-@pytest.mark.xfail(raises=ValueError)
-def test_raises_for_invalid_geojson():
-    load_geojson_file.cache_clear()
-    with mock.patch(
-        'notifications_utils.broadcast_areas.Path.read_text',
-        return_value='{"a": 1}',
-    ):
-        BroadcastAreaLibraries()
-
-
 def test_loads_areas_from_library():
     assert [
         (area.id, area.name) for area in sorted(
             broadcast_area_libraries.get('countries')
         )
     ] == [
-        ('england', 'England'),
-        ('northern-ireland', 'Northern Ireland'),
-        ('scotland', 'Scotland'),
-        ('wales', 'Wales'),
+        ('countries-E92000001', 'England'),
+        ('countries-N92000002', 'Northern Ireland'),
+        ('countries-S92000003', 'Scotland'),
+        ('countries-W92000004', 'Wales'),
     ]
 
```

Comment:
Is it worth adding a similar test for some wards? Just to check that all the wards for a given area have the names and IDs we’d expect?